enterprise/coderd/licenses_internal_test.go

```diff
@@ -0,0 +1,153 @@
+package coderd
```

Comment:
It seems like this test doesn't have to be internal.

Reply:
it modifies the `keys` variable in-place during the tests (patch paradigm).  I want to keep that variable unexported, so the tests need to be internal.